[15.0][MIG] sale project service tracking copy tasks (#1188)
OCA/project: sale_project_service_tracking_copy_tasks/readme/DESCRIPTION.rst

> This module add the possibility to create task in an existing project from the sales order through setting the product with a project template with the desired task.

Reviewer: ```suggestion
This module allows creating tasks in an existing project from the sales order through setting the product with a project template with the desired task.
```

> The use case is when a Odoo customer sell the same services with the same task constantly and the task should be linked with existing project. Odoo don't cover this use case as Odoo always create a new project.

Reviewer: **nitpick:** add this to a `CONTEXT.rst` file instead.

```suggestion
The use case is when an Odoo customer sells the same services with the same tasks constantly and the tasks should be linked with existing project. Standard Odoo would always create a new project.
```
